runtime/lib/ttnn/operations/data_movement/transpose.cpp

```diff
@@ -35,6 +35,7 @@ void run(const ::tt::target::ttnn::TransposeOp *op, ProgramContext &context) {
       utils::createMemoryConfig(op->out());
```

Comment:
>   // Ideally this would use ttnn::transpose, but since ttnn::transpose doesn't
  // work at the moment, we use this temporary solution.

@sdjordjevicTT just FYI. Seems like one more issues for ttnn folks

Reply:
I think @vladimirjovanovicTT had some context as to why this wasn't working. We should create an issue on their side!

Reply:
@vladimirjovanovicTT once you see this PR, give us more context and let's create followup issue on tt-metal as well :))

Reply:
I currently don't remember the details regarding ttnn::transpose, but my comments would be obsolete since squeeze from 4D also didn't work in the TTNN version that was being used back then (that's why it wasn't added). Does transpose work now @dgolubovicTT ?
Anyways, we should test current ttnn::transpose behavior with the current version before raising an issue on their side.

Reply:
Transpose works now, since I added squeeze back to original dimensions... I will check if ttnn::transpose works on its own...